Source/Blazorise/Components/Table/TableRowGroup.razor.cs

```diff
@@ -49,10 +50,21 @@ private void BuildRowCellClasses( ClassBuilder builder )
         builder.Append( ClassProvider.TableRowGroupCell() );
     }
 
+    /// <summary>
+    /// Builds the classnames for an indent cell.
+    /// </summary>
+    /// <param name="builder">Class builder used to append the classnames.</param>
+    private void BuildRowIndentCellClasses( ClassBuilder builder )
+    {
+        builder.Append( ClassProvider.TableRowGroupCell() );
```

Comment:
I think TableRowGroupCell should not be here.

Reply:
I could see it being there, it's still a regular group cell. But I'll remove then.